CHANGES.md

```diff
@@ -1,3 +1,13 @@
+## edge-19.9.5
+
+This edge release is a release candidate for `stable-2.6`.
+
+* CLI
+  * Added headers to the JSON output for `linkerd tap`
```

Comment:
I know most readers will understand the meaning of "headers", but I think something like:

> Added request and response headers to the JSON output option for `linkerd tap`

provides a little more detail about what headers have actually been added (and where to look!)